### PR TITLE
Feature/federate prom

### PR DESCRIPTION
**Pull request by Judunet**
New feature to include the multiple iceberg architecture.

### CODE REVIEW

File: roles/addons/prometheus_server/tasks/main.yml [hunk 1]
```diff
@@ -44,6 +44,20 @@
   tags:
     - firewall
 
+- name: "copy █ Install karma firewalld service configuration file"
```
Comment (btravouillon) [inline roles/addons/prometheus_server/tasks/main.yml:47]: I would copy both files alertmanager.xml and karma.xml in the same task, with a loop.

Off the top of my head:
```
copy:
  src: {{ item }}
  dest:  "/etc/firewalld/services/{{ item }}"
[...]
loop: ['alertmanager.xml', 'karma.xml']
```

File: roles/addons/prometheus_server/handlers/main.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
 ---
 - name: reload firewalld
   command: firewall-cmd --reload
+
+- name: service Restart prometheus server
+  service:
+    name: prometheus
+    state: restarted
```
Comment (btravouillon) [inline roles/addons/prometheus_server/handlers/main.yml:8]: Can you add:
```
 when:
    - "'service' not in ansible_skip_tags"
    - (start_services | bool)
```

**Review by btravouillon — COMMENTED** [review]
Hi @Judunet,

I started the review but have a question before going further. Is Prometheus federation compatible with Thanos?

**Comment by btravouillon** [comment]
>  For Thanos i think it use federation, i will check that

Were you able to check that?

**Comment by oxedions** [comment]
Dear @Judunet 

Any news on that matter ?

**Comment by oxedions** [comment]
Closing for now.

**Pull request closed without merging**